client/src/pages/game_lobby/GameLobby.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+
+import { copyToClipboard } from "../../utils/utils";
+import Header from "../common/Header";
+import TeamSelect from "./team_select/TeamSelect";
+import "../common/common.css";
+import "./GameLobby.css";
+
+function GameLobby() {
+  const currentUser = { sessionID: 1, firstName: "Tony" }; // dummy data of current user. sessionID will be generated from web socket in backend
+  const [matchID] = useState("ABCD"); // dummy match ID
+  const [canStartGame, setCanStartGame] = useState(false);
+
+  const isTeamReady = (team) => {
+    const SPYERMASTER_INDEX = 0;
```

Comment:
since these variables never change, you can consider moving them outside of this function. great for modularity and also could be a little bit more space efficient